Helper/Tracking.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Created by PhpStorm.
```

Comment:
are these needed? why are they only on some files?

Reply:
they're not required (there is a lint rule for it but I turned it off because it was too noisy IMO). They're on some files because I happened to use phpstorm to create them back in 2017 (this looks new because it's been renamed/refactored)